test/net/sourceforge/kolmafia/FamiliarDataTest.java

```diff
@@ -154,4 +167,56 @@ public void canHandleGreyGooseCombatSkills() {
     assertFalse(KoLCharacter.hasCombatSkill(SkillPool.CONVERT_MATTER_TO_ENERGY));
     assertFalse(KoLCharacter.hasCombatSkill(SkillPool.CONVERT_MATTER_TO_POMADE));
   }
+
+  @Nested
+  class API {
+    @AfterEach
```

Comment:
`@BeforeEach`, or possibly `@BeforeAll`.

Reply:
BeforeAll. My previous experience was that you couldn't use BeforeAll and AfterAll in Nested test classes.
I was wrong. It does work.

There are other tests in this package which add familiars to KoLCharacter - such as the Fidoxene tests - and before I made this API test clean up after itself, they started failing - because of leakage.